### PR TITLE
Improve code style and docs of job

### PR DESCRIPTION
**Pull request by sauloperez**
#### What? Why?

Given that I was investigating the "notify producers" button that shows up in the order cycle page, I thought I could take the chance to improve a bit the code while I was reading it. That's all.

#### What should we test?

Nothing. There's no behavior change at all.

### CODE REVIEW

File: app/mailers/producer_mailer.rb [hunk 1]
```diff
@@ -12,12 +12,14 @@ def order_cycle_report(producer, order_cycle)
 
     subject = "[#{Spree::Config.site_name}] #{I18n.t('producer_mailer.order_cycle.subject', producer: producer.name)}"
 
-    if has_orders? order_cycle, producer
-      mail(to: @producer.email,
-           from: from_address,
-           subject: subject,
-           reply_to: @coordinator.email,
-           cc: @coordinator.email)
+    if has_orders?(order_cycle, producer)
```
Comment (sauloperez) [inline app/mailers/producer_mailer.rb]: I think it's a bit too late to check whether the order cycle has orders. It's pointless to send a notification if there are none. IMO we should check before enqueuing the `OrderCycleNotificationJob`. I'll create an issue for it.

File: app/jobs/order_cycle_notification_job.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
+# Deliers an email with a report of the order cycle to each of its suppliers
```
Comment (enricostano) [inline app/jobs/order_cycle_notification_job.rb]: `s/Deliers/Delivers/`

File: app/mailers/producer_mailer.rb (hunk 1 above)
Comment (sauloperez) [reply app/mailers/producer_mailer.rb]: done

**Comment by daniellemoorhead** [comment]
Hey there lovely Katuma people. 

Would suggest staging this and checking that it does indeed continue to work as expected. If it doesn't break anything then go ahead and merge it (don't forget test notes!).

#myfirstgifof2018
giphy

**Comment by kirstenalarsen** [comment]
made me laugh out loud

**Comment by sauloperez** [comment]
The `OrderCycleNotificationJob` is successfully triggered in both staging.katuma.org and development. Working!

**Pull request merged**